build_scripts/build_usd.py

```diff
@@ -201,14 +201,29 @@ def RunCMake(context, force, extraArgs = None):
 
     generator = context.cmakeGenerator
 
+    # OpenSubdiv seems to error when building on windows w/ Ninja...
+    # ...so just use the Visual Studio generator until someone can
+    # sort it out
+    if (generator == "Ninja" and Windows() and
+            context.currentDependency.name == "OpenSubdiv"):
+        generator = None
+
     # On Windows, we need to explicitly specify the generator to ensure we're
     # building a 64-bit project. (Surely there is a better way to do this?)
+    # TODO: figure out exactly what "vcvarsall.bat x64" sets to force x64
     if generator is None and Windows():
         if IsVisualStudio2017OrGreater():
             generator = "Visual Studio 15 2017 Win64"
         else:
             generator = "Visual Studio 14 2015 Win64"
 
+    if generator == 'Ninja':
```

Comment:
For simplicity, instead of branching can we just always do:
```
cmakeBuildType = '-DCMAKE_BUILD_TYPE=Release '
buildConfig = '--config Release '
```
or does the Ninja generator not like the '--config Release' bit?

Reply:
I think Ninja will just safely ignore it, so made it non-branching as requested...